ReactNativeClient/lib/shim-init-node.js

```diff
@@ -12,6 +12,14 @@ const { _ } = require('lib/locale.js');
 const http = require('http');
 const https = require('https');
 const toRelative = require('relative');
+const { Database } = require('lib/database.js');
+
+let keytar;
+try {
+	keytar = require('keytar');
+} catch (err) {
+	keytar = null;
+}
```

Comment:
`keytar` is installed and always available so no need for a try/catch block

Reply:
The reason I used a try/catch block here is because when the system doesn't have a keychain, keytar threw an error, even when keytar itself was installed.

Reply:
Really? Ok that's not great, but in that case your implementation indeed makes sense. Thanks for looking into it.

Reply:
Actually please could you add a comment then about your try/catch block to explain why it's there?

Reply:
Yes, sure.